sdk/python/feast/client.py

```diff
@@ -977,197 +601,128 @@ def ingest(
                     * csv
                     * json
 
+            project: Feast project to locate FeatureTable
+
             chunk_size (int):
                 Amount of rows to load and ingest at a time.
 
             max_workers (int):
                 Number of worker processes to use to encode values.
 
-            disable_progress_bar (bool):
-                Disable printing of progress statistics.
-
             timeout (int):
                 Timeout in seconds to wait for completion.
 
-        Returns:
-            str:
-                ingestion id for this dataset
-
         Examples:
             >>> from feast import Client
             >>>
             >>> client = Client(core_url="localhost:6565")
-            >>> fs_df = pd.DataFrame(
+            >>> ft_df = pd.DataFrame(
             >>>         {
             >>>            "datetime": [pd.datetime.now()],
             >>>            "driver": [1001],
             >>>            "rating": [4.3],
             >>>         }
             >>>     )
             >>> client.set_project("project1")
-            >>> client.ingest("driver", fs_df)
             >>>
-            >>> driver_fs = client.get_feature_set(name="driver", project="project1")
-            >>> client.ingest(driver_fs, fs_df)
+            >>> driver_ft = client.get_feature_table("driver")
+            >>> client.ingest(driver_ft, ft_df)
         """
 
-        if isinstance(feature_set, FeatureSet):
-            name = feature_set.name
-            project = feature_set.project
-        elif isinstance(feature_set, str):
-            if self.project is not None:
-                project = self.project
-            else:
-                project = "default"
-            name = feature_set
-        else:
-            raise Exception("Feature set name must be provided")
+        if project is None:
+            project = self.project
+        if isinstance(feature_table, FeatureTable):
+            name = feature_table.name
 
         # Read table and get row count
-        dir_path, dest_path = _read_table_from_source(source, chunk_size, max_workers)
-
-        pq_file = pq.ParquetFile(dest_path)
-
-        row_count = pq_file.metadata.num_rows
+        dir_path, dest_path, column_names = _read_table_from_source(
+            source, chunk_size, max_workers
+        )
 
         current_time = time.time()
-
-        print("Waiting for feature set to be ready for ingestion...")
+        print("Waiting for feature table to be ready for ingestion...")
```

Comment:
What are you waiting for here? There is no status on a feature table

Reply:
Removed.